### DISCUSSION TITLE
Welcome to plugin.video.rtsplaytv Discussions!

### DISCUSSION THREAD

**Discussion by goggle** [goggle/plugin.video.rtsplaytv#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gaet81** [comment]
Hi,

First of all, thanks for this great plugin!
In the feature it says that the plugin contains livestream but I don't see them in the interface, can you tell me where can I find them?

Thanks,

Gaëtan

**Reply by goggle** [reply]
Go to `Topics -> Sports -> Sports en direct et prochains directs`
